client/js/widgets/widget.js

```diff
@@ -6,7 +6,6 @@ import { showOverlay, shuffleWidgets, sortWidgets } from '../main.js';
 import { tracingEnabled } from '../tracing.js';
 import { toHex } from '../color.js';
 import { center, distance, overlap, getOffset, getElementTransform, getScreenTransform, getPointOnPlane, dehomogenize, getElementTransformRelativeTo } from '../geometry.js';
-import { defaultPileSnapRange } from './pile.js';
```

Comment:
I think the fact that I needed to remove this might be a bundler error, that even though modules support bidirectional dependencies it uses the imports to establish a relative order in the bundle. If we wanted to keep the imports pure I could resolve this by putting the constant in a different file or perhaps some other method of determining the pile's default `pileSnapRange`.

The current patch works, but happy to go with some other approach if desired.